governance.md

```diff
@@ -24,17 +28,53 @@ Every code owner is expected to:
 * Approve only those PRs in which the contributor made the requested improvements
 * Check if the related CI tests have completed successfully before approving the PR
 * Make sure that the PR approval flow runs smoothly
-* Prioritize issues and manage their processing
 * Proactively fix bugs
 * Perform maintenance tasks for their projects
-
-### Add or remove a code owner
+* Issue that is assigned to current release is treated with priority by maintainer that is an assignee
+* Monitor Slack
+* Triage GitHub issues and perform pull request reviews for other maintainers and the community. Follow [labels workflow](/governance/#issues-workflow-issues-workflow) document
+* In general continue to be willing to spend at least 50% of one's time working on Kyma (~2.5 business days per week).
+
+### Process for becoming a maintainer aka codeowner
+
+* Express interest to the existing maintainers that you or your organization are interested in becoming a
+  maintainer. The difference is that in case you are individual or corporate contributor/maintainer then different contributor license agreement applies.
+* Express what are the areas of Kyma that you want to engage with. Kyma, because of its goal, is complex and it is impossible to become a maintainer in every area. Make sure you get support from proper group of maintainers.
+* Becoming a maintainer generally means that you are going to be spending substantial
+  time (>50%) on Kyma for the foreseeable future.
+* We will expect you to start contributing increasingly complicated PRs, under the guidance
+  of the existing maintainers.
+* We may ask you to do some PRs from our backlog.
+* As you gain experience with the code base and our standards, we will ask you to do code reviews for incoming PRs (i.e., all maintainers are expected to shoulder a proportional share of community reviews).
+* After a period of approximately 3 months of working together and making sure we see eye to eye, the existing maintainers will confer and decide whether to grant maintainer status or not. We make no guarantees on the length of time this will take, but no longer than 3 months.
```

Comment:
IMO it will be more convenient that contributor will open an issue to be added as maintainer instead of requiring it from current maintainers. 

In the given issue he should "document" his current contributions and then maintainers should vote if he should be added or not.